src/schemas/components/SchemaFormProperties.vue

```diff
@@ -65,13 +68,30 @@
   }
 
   function setValue(propertyKey: string, value: unknown): void {
-    emit('update:values', {
-      ...props.values,
-      [propertyKey]: value,
-    })
+    patches.push({ propertyKey, value })
+
+    flush()
   }
 
   function getRequired(propertyKey: string): boolean {
     return props.parent.required?.includes(propertyKey) ?? false
   }
+
+  const patches: { propertyKey: string, value: SchemaValue }[] = []
+
+  const flush = debounce(() => {
+    const updatedValues = { ...props.values }
+
+    patches.forEach(({ propertyKey, value }) => {
+      updatedValues[propertyKey] = value
+
+      if (isNullish(value)) {
+        delete updatedValues[propertyKey]
+      }
```

Comment:
This make sure that null/undefined values don't appear in the form values. That way we won't get errors like `'null' is not 'int'`.